### PR TITLE
Added details and images to my interests section.

### PR DESCRIPTION
**Pull request by imanherzi**
Added details and images to my interests section.

Added 4 images and a small blurb about my interests.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,16 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function generateRandomFact() {
+  const facts =
+      ['I read over 150 books last year.', 'I started programming 2 years ago.', 'I did a painting every day for a year.', 'I\'m a dual citizen of Canada and the United States.'];
```
Comment (tambchop) [inline portfolio/src/main/webapp/script.js:20]: Every day for a year?  =o  Impressive commitment!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,66 @@
...
+        <section id="random-fact">
+          <div id="fact-container"></div>
+          <button onclick="generateRandomFact()">random fact</button>
+        </section>
+    
+      </section>
+
+      <section>
+        <h3>My Interests</h3>
+        <p>
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html:34]: Suggest making this a div instead of p

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,66 @@
...
+          I enjoy programming and I especially enjoy web development because it's so much fun to interact with the end result.
+          In my free time I like to read, paint, and play video games. Here are a few random pictures of things I love.
+          
+          <figure>
+            <img src="/images/malcolm-tucker.jpg" width="500px" alt="A captioned photo from the Thick of It (2005) of Malcom Tucker
```
Comment (kzweller) [inline portfolio/src/main/webapp/index.html:39]: Suggest not in-lining CSS (e.g. "width=X")... Instead put in CSS.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (imanherzi) [reply portfolio/src/main/webapp/script.js:20]: :D

**Pull request merged**